docs/project/2024/rares.ionescu3004/index.md

```diff
@@ -0,0 +1,112 @@
+| [LEDs (different colors)](https://www.farnell.com/datasheets/1498852.pdf) | Using different color LEDs we can display different color combinations indicating system messages | [0.5 RON per LED](https://ardushop.ro/ro/electronica/299-led-5mm.html?search_query=LED&results=242) |
+| [RGB LED](https://www.farnell.com/datasheets/3497864.pdf) | Depending on its color wo know the state of the system | [5 RON](https://ardushop.ro/ro/electronica/271-led-tricolor-cu-catod-comun.html?search_query=RGB+LED&results=249) |
+| [Buzzer](https://components101.com/misc/buzzer-pinout-working-datasheet) | Goes off when an anomaly is detected | [4 RON](https://ardushop.ro/ro/electronica/194-buzzer.html?search_query=buzzer&results=16) |
+| [DC Motor with Propeller](https://ardushop.ro/ro/motoare-si-drivere/437-motoras-curent-continuu.html) | Should run at some exact parameters set by the Facility Pico | [2 RON](https://ardushop.ro/ro/motoare-si-drivere/437-motoras-curent-continuu.html) |
+| [Jumper Wires](https://media.digikey.com/pdf/Data%20Sheets/Digi-Key%20PDFs/Jumper_Wire_Kits.pdf) | Connect everything together | [8 RON](https://www.optimusdigital.ro/en/wires-with-connectors/12-breadboard-jumper-wire-set.html) |
+| [Potentiometer](https://www.handsontec.com/dataspecs/passive/WH148%20Pot-meter.pdf) | Should be used to change the input | [2 RON](https://www.optimusdigital.ro/en/potentiometers/901-10k-wh148-variable-resistor.html) |
+| [7-Segment Display](https://www.sparkfun.com/datasheets/Components/LED/7-Segment/YSD-439AR6B-35.pdf) | Will be used to display different data or status codes | [4 RON](lhttps://ardushop.ro/ro/electronica/191-display-led-4x7-segmente.html?search_query=Display+led+7+segmente&results=312) |
+| [Buttons](https://components101.com/switches/push-button) | Used to select the type of attack | [0.63 RON](https://ardushop.ro/ro/home/97-buton-mic-push-button-trough-hole.html?search_query=push+button&results=30) |
+| [Breadboard](https://components101.com/sites/default/files/component_datasheet/Breadboard%20Datasheet.pdfn) | To connect different components onto it | [10 RON](https://ardushop.ro/ro/electronica/33-breadboard-830.html?search_query=breadboard&results=31) |
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```

Comment:
Do you need these libraries?

Reply:
Yes, I am planning to use both libraries for my project for the explorer base.